requirements/production.txt

```diff
@@ -2,4 +2,4 @@ docutils>=0.12
 Jinja2>=2.7.3
 MarkupSafe>=0.23
 Pygments>=1.6
-Sphinx>=4.0,<6,!=5.0.0
+Sphinx>=4.0,<7,!=5.0.0
```

Comment:
```suggestion
Sphinx>=4.0,!=5.0.0
```
The release cadence of Sphinx has changed so major versions may now happen more often than once a year. I suggest we don't upper bound the version as we would have to be much more vigilant and update Breathe.

Reply:
Thanks @jakobandersen - should we do this in setup.py too? 

Cheers for reviewing it. I am very aware that you normally handle this but felt maybe you were busy. I should have @-ed you anyway :)

Reply:
> should we do this in setup.py too?

Yes, I forgot about that having to be modified as well.

> Cheers for reviewing it. I am very aware that you normally handle this but felt maybe you were busy. I should have @-ed you anyway :)

No problem, I have indeed been too busy to handle it, so great you have done so :-).